integration/main_test.go

```diff
@@ -79,8 +79,10 @@ func TestMain(m *testing.M) {
 // setup can be used to initialize things before integration tests start (as of now it only builds the services used by the integration tests so they can be referenced)
 func setup() ([]func() error, error) {
 	var (
-		serviceName = "testing"
-		targetStage = "containerd-snapshotter-base"
+		serviceName   = "testing"
+		targetStage   = "containerd-snapshotter-base"
+		registryStage = "registry2"
```

Comment:
Sorry, can we update these names too? I don't have a strong preference - `registry2Stage` and `registry3alpha1Stage`? `defaultRegistryStage` and `artifactRegistryStage`?